openwisp_radius/signals.py

```diff
@@ -0,0 +1,3 @@
+from django.dispatch import Signal
+
+radius_accounting_success = Signal(providing_args=['accounting_data'])
```

Comment:
`providing_args` doesn't play well with django 4 in which it was removed, please see the release notes of django 4 (lookfor `providing_args`) and update this code according to what is suggested there.